salt/orchestrate/aws/map_templates/edx_worker.yml

```diff
@@ -0,0 +1,24 @@
+edx_worker:
+  {% set num_subnets = subnetids|length %}
```

Comment:
We should just include the worker nodes in the same map template as the edX instances so that creation of all of them can be executed in parallel.